content/en/docs/tasks/administer-cluster/use-cascading-deletion.md

```diff
@@ -0,0 +1,159 @@
+
+The output has an `ownerReferences` field similar to this:
+
+```
+apiVersion: v1
+    ...
+    ownerReferences:
+    - apiVersion: apps/v1
+      blockOwnerDeletion: true
+      controller: true
+      kind: ReplicaSet
+      name: nginx-deployment-6b474476c4
+      uid: 4fdcd81c-bd5d-41f7-97af-3a3b759af9a7
+    ...
+```
+
+## Use foreground cascading deletion
+
+By default, Kubernetes uses [background cascading deletion](/docs/concepts/workloads/controllers/garbage-collection/#background-deletion)
+to delete dependents of an object. You can switch to foreground cascading deletion
+using either `kubectl` or the Kubernetes API. 
+
+{{<tabs name="foreground_deletion">}}
+{{% tab name="kubectl" %}}
+You can delete objects using foreground cascading deletion with `kubectl` if
+your cluster is running Kubernetes version 1.20.x or later.
+
+<!--TODO: verify release after which the --cascade flag is switched to a string in https://github.com/kubernetes/kubectl/commit/fd930e3995957b0093ecc4b9fd8b0525d94d3b4e-->
+
+```shell
+kubectl delete deployment nginx-deployment --cascade=foreground
+```
+{{% /tab %}}
+{{% tab name="API" %}}
+You can delete objects using foreground cascading deletion by calling the
+Kubernetes API.
+
+1. Start a local proxy session:
+
+    ```shell
+    kubectl proxy --port=8080
+    ```
+
+1. Use `curl` to trigger deletion:
+
+   ```shell
+   curl -X DELETE localhost:8080/apis/apps/v1/namespaces/default/deployments/nginx-deployment \
+       -d '{"kind":"DeleteOptions","apiVersion":"v1","propagationPolicy":"Foreground"}' \
+       -H "Content-Type: application/json"
+   ```
+
+   The output contains a `foregroundDeletion` {{<glossary_tooltip text="finalizer" term_id="finalizer">}}
+   like this:
+
+   ```
+    "kind": "Deployment",
+    "apiVersion": "apps/v1",
+    "metadata": {
+        "name": "nginx-deployment",
+        "namespace": "default",
+        "uid": "d1ce1b02-cae8-4288-8a53-30e84d8fa505",
+        "resourceVersion": "1363097",
+        "creationTimestamp": "2021-07-08T20:24:37Z",
+        "deletionTimestamp": "2021-07-08T20:27:39Z",
+        "finalizers": [
+          "foregroundDeletion"
+        ]
+        ...
+   ```
+   {{% /tab %}}
+   {{</tabs>}}
+
+## Use background cascading deletion
+
+1. [Create a sample Deployment](/docs/tasks/run-application/run-stateless-application-deployment/#creating-and-exploring-an-nginx-deployment).
+1. Use either `kubectl` or the Kubernetes API to delete the Deployment.
+
+{{<tabs name="background_deletion">}}
+{{% tab name="kubectl" %}}
+Kubernetes uses background cascading deletion by default, and will do so
+even if you run the following commands without the `--cascade` flag.
+
+**Kubernetes version 1.20.x and later**
+
+Run the following command:
+
+```shell
+kubectl delete deployment nginx-deployment --cascade=background
+```
+
+**Kubernetes versions prior to 1.20.x**
+
+Run the following command:
+
+```shell
+kubectl delete deployment nginx-deployment --cascade=true
+```
```

Comment:
Having seen the preview, I'd prefer to make this Kubernetes version choice happen in tabs, and use headings to separate the `kubectl` vs. API + curl approaches.

The second tab can also refer readers to https://kubernetes.io/docs/home/supported-doc-versions/

Reply:
Okay. Can you confirm what the actual version number is? I wasn't sure from the commit (I've added a TODO comment in the MD to remind myself to verify)

Reply:
I don't know when the change happened (I'd have to ask Google or, failing that, spelunk in the source code history for `kubectl`).